src/integer/z/sample/uniform.rs

```diff
@@ -142,3 +206,100 @@ mod test_sample_uniform {
         }
     }
 }
+
+#[cfg(test)]
+mod test_sample_prime_uniform {
+    use crate::{integer::Z, integer_mod_q::Modulus};
+    use std::str::FromStr;
+
+    /// Checks whether `sample_prime_uniform` outputs a prime sample every time.
+    #[test]
+    fn is_prime() {
+        for _ in 0..8 {
+            let sample = Z::sample_prime_uniform(20, i64::MAX).unwrap();
+            assert!(sample.is_prime());
```

Comment:
maybe add a comment in the assert, that it can potentially fail, even though only with very small probability